code/menuui/barracks.cpp

```diff
@@ -433,7 +433,7 @@ void barracks_init_stats(scoring_struct *stats)
 	Num_stat_lines++;
 
 	Assert(Num_stat_lines < Max_stat_lines);
-	STRCPY1(Stat_labels[Num_stat_lines], "Current Score:");
+	STRCPY1(Stat_labels[Num_stat_lines], XSTR( "Current Score:", 1583));
```

Comment:
You can just replace `STRCPY1` with `strcpy_s` here to fix the CI failure.

Reply:
Well, i have to admit, that this has irritated me a bit, because STRCPY1 is used for all other XSTRs in that area well and it only complains about this one. Will change that, of course.

Reply:
The check that showed the error message only runs on changed lines so that is the reason why it didn't show any errors on any of the other `STRCPY1` calls.